service.py

```diff
@@ -0,0 +1,53 @@
+from concurrent import futures
+import grpc
+import time
+import utility
+import hwsc_file_transaction_svc_pb2
+import hwsc_file_transaction_svc_pb2_grpc
+
+class FileTransactionService(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer):
+     def __init__(self):
+
+         class Servicer(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer):
+             def __init__(self):
+                 pass
+
+             def get_status(self, request, context):

```

Comment:
Need follow up changes referring to https://github.com/hwsc-org/hwsc-api-blocks/pull/89#discussion_r238925346